src/WebJobs.Script/Description/Node/functions/functions.js

```diff
@@ -39,7 +39,11 @@ function createFunction(f) {
 
         context.done = function (err, result) {
             if (context._done) {
-                context.log("Error: 'done' has already been called. Please check your script for extraneous calls to 'done'.");
+                if (context._promise) {
+                    context.log("Error: Choose either to return a promise or call 'context.done'.  Do not use both in your script.");
+                } else {
+                    context.log("Error: 'done' has already been called. Please check your script for extraneous calls to 'done'.");
```

Comment:
Let's make these error messages consistent by changing this one to reference 'context.done' as well. However, context is actually a variable name that users can change, so perhaps both error messages should just say 'done'?